x-pack/plugins/aiops/public/components/log_categorization/log_categorization_for_embeddable/minimum_time_range.ts

```diff
@@ -7,6 +7,8 @@
 import { i18n } from '@kbn/i18n';
 import type { unitOfTime } from 'moment';
 
+export const DEFAULT_MINIMUM_TIME_RANGE_OPTION: MinimumTimeRangeOption = 'No minimum';
```

Comment:
I think we could do with some clarification in the UI as to what 'No minimum' means? Possibly some text in the help tooltip, and some text below the 'Minimum time range' field if they have 'No minimum' selected. e.g. does it mean it will search right back to the start of the data in the data view, or just to the start date set in the time picker.

<img width="464" alt="Screenshot 2024-08-23 at 15 56 01" src="https://github.com/user-attachments/assets/8f43b6de-a746-43ac-90e6-0a909de1e8a3">

Reply:
Yeah I agree. I'm not sure what the text should say for the `No minimum` option.
`The time range used will match the time picker`  ?
`No additional time will be added to the range selected` ?

@szabosteve do you have any ideas?

Reply:
@jgowdyelastic @peteharverson
I'd change the dropdown label to `Use selected time`.
And the helper text – if needed – could be something like:
`The time range you specified using the time selector will be used for the analysis.`
What do you think?

Reply:
I like the idea of changing the label in the dropdown from 'No minimum' to 'Use selected time' (or even 'Use range selected in time picker' to clarify which selected time?).

The only issue with `The time range you specified using the time selector will be used for the analysis.` for the help text is that it then begs the question of why wouldn't the analysis always used the selected time range? Maybe something along the lines of `No additional time be added.` ? 

Very difficult to explain concisely what this setting relates to!

Reply:
What about the following as the helper text?
`No additional time is added to the range you specified with the time selector.`

However, if the selector label is changed to `Use range specified in time selector` or something along these lines, then I don't think a helper text is necessary.

Reply:
Text updated to:
![image](https://github.com/user-attachments/assets/578dbd35-6759-4e5a-9def-e661bb21ce25)

Reply:
Nit - I think the form help texts normally end with a `.`